.azure-pipelines/azure-pipelines-create-release.yml

```diff
@@ -46,7 +46,7 @@ jobs:
   steps:
   - template: templates/run-tests.yml
     parameters:
-      pythonVersion: '3.7.0'
```

Comment:
You will have to fix this in the release branch (release 0.4.0) also. 
For [this](https://dev.azure.com/ms/azure-devops-cli-extension/_build?definitionId=36) pipeline.

Reply:
I think you already fixed that in your PR #529 for Ubuntu machines. And we don't use any Mac machines in [release pipeline](https://dev.azure.com/ms/azure-devops-cli-extension/_build?definitionId=36)

Reply:
My bad! This is create release yml. I just saw *release.yml and thought this is the same yml file.